ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -626,7 +627,7 @@ func (r *RayClusterReconciler) updateStatus(instance *rayiov1alpha1.RayCluster)
 	// validation for the RayStartParam for the state.
 	isValid, err := common.ValidateHeadRayStartParams(instance.Spec.HeadGroupSpec)
 	if err != nil {
-		r.Recorder.Event(instance, v1.EventTypeWarning, "Parameters conflict", err.Error())
+		r.Recorder.Event(instance, v1.EventTypeWarning, rayevents.Unhealthy, err.Error())
```

Comment:
Is it possible to keep the specificity of "Parameters conflict" (vs "RayUnhealthy")

Reply:
If https://github.com/ray-project/kuberay/pull/332/files#diff-e72304540ddff812f22838c9f50e648bf9d7247c43bba7f4c5b00739e0f15cbdR589-R593 is merged, the `err` here will contain the more detailed error message `RayStartParams: object store memory exceeds ...` in the event message. Seems like convention is to keep event reason one word, came-cased. Lmk what you think.

Reply:
Ah, got it -- details come from err.